### ISSUE TITLE
Notebook output line spacing is incorrect

### ISSUE THREAD

**Issue by ortk95** [microsoft/vscode#142834]
Issue Type: <b>Bug</b>

Notebook output cells have a large amount of space between each line which is inconsistent with the terminal and breaks any complex output such as box drawing characters and text on a coloured background:

![image](https://user-images.githubusercontent.com/26504228/153586963-df77bed9-b7e2-49df-876f-8a6ee7c5743d.png)

<details>
    <summary>Code to reproduce image</summary>

    for s in ['This text should have ', 'a solid red background']:
        print(f'\x1b[41m{s}\x1b[0m')
    
    print("""
    ┏━━━━━━━━━━━━━━━━┓
    ┃This text should┃
    ┃be in a nice box┃
    ┗━━━━━━━━━━━━━━━━┛""")
    
    # From https://en.wikipedia.org/wiki/Box-drawing_character
    print("""
    ┌─┬┐  ╔═╦╗  ╓─╥╖  ╒═╤╕
    │ ││  ║ ║║  ║ ║║  │ ││
    ├─┼┤  ╠═╬╣  ╟─╫╢  ╞═╪╡
    └─┴┘  ╚═╩╝  ╙─╨╜  ╘═╧╛
    ┌───────────────────┐
    │  ╔═══╗ Some Text  │▒
    │  ╚═╦═╝ in the box │▒
    ╞═╤══╩══╤═══════════╡▒
    │ ├──┬──┤           │▒
    │ └──┴──┘           │▒
    └───────────────────┘▒
     ▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒""")
    
</details>

Almost all other command line type interfaces have zero line spacing for the output text, so this makes VS code notebooks the only place that has this kind of issue.

It would be great if the notebook output line spacing was at least customisable so that more complex printed output could be displayed as expected without the ugly space between each line.

VS Code version: Code 1.64.1 (d6ee99e4c045a6716e5c653d7da8e9ae6f5a8b03, 2022-02-07T17:26:08.977Z)
OS version: Darwin x64 20.6.0
Restricted Mode: No

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i5-7360U CPU @ 2.30GHz (4 x 2300)|
|GPU Status|2d_canvas: enabled<br>gpu_compositing: enabled<br>metal: disabled_off<br>multiple_raster_threads: enabled_on<br>oop_rasterization: enabled<br>opengl: enabled_on<br>rasterization: enabled<br>skia_renderer: disabled_off_ok<br>video_decode: enabled<br>webgl: enabled<br>webgl2: enabled|
|Load (avg)|9, 18, 18|
|Memory (System)|16.00GB (0.28GB free)|
|Process Argv|--crash-reporter-id 34d9085e-a874-4cbf-a2f4-655c27ce98d5|
|Screen Reader|no|
|VM|0%|
</details><details><summary>Extensions (18)</summary>

Extension|Author (truncated)|Version
---|---|---
Bookmarks|ale|13.2.3
rtf|ale|2.4.0
dhall-lang|dha|0.0.4
gitlens|eam|11.7.0
vscode-back-forward-button|gri|0.1.6
crontab-syntax-highlight|hog|0.0.2
latex-workshop|Jam|8.23.0
csharp|ms-|1.24.0
python|ms-|2022.0.1814523869
vscode-pylance|ms-|2022.2.1
jupyter|ms-|2022.1.1001821375
jupyter-keymap|ms-|1.0.0
jupyter-renderers|ms-|1.0.6
mono-debug|ms-|0.16.2
vscode-yaml|red|1.4.0
multi-command|ryu|1.5.1
latex-support|tor|4.0.0
vscodeintellicode|Vis|1.2.17

</details><details>
<summary>A/B Experiments</summary>

```
vsliv368:30146709
vsreu685:30147344
python383:30185418
vspor879:30202332
vspor708:30202333
vspor363:30204092
pythonvspyl392:30425749
pythontb:30283811
pythonptprofiler:30281270
vshan820:30294714
vstes263:30335439
vscorecescf:30435083
pythondataviewer:30285071
vscod805:30301674
pythonvspyt200:30340761
binariesv615:30325510
bridge0708:30335490
bridge0723:30353136
vsaa593:30376534
vsc1dst:30433059
pythonvs932:30410667
wslgetstartedc:30433508
vsrem710cf:30416617
vscscmwlcmt:30433761

```

</details>

**Comment by rebornix** [comment]
we might need to introduce line height setting for output

**Comment by lakshits11** [comment]
Yes please introduce a line height feature for this. I also opened up a issue #142387 but no one responded.

**Comment by agnz** [comment]
This would be extremely helpful!

**Comment by ortk95** [comment]
I have found a temporary workaround which is to run the following command in the notebook interactive window:
```python
from IPython.core.display import HTML; HTML(r'<style> .output-plaintext, .output-stream, .output { font-family: Menlo; line-height: 1.2em !important; } </style>')
```
This sets the line height (and font) for the lifetime of the current interactive window, but has to be run manually every time a new interactive window is created. I tried adding the command to `jupyter.runStartupCommands` but this does not seem to have an effect unfortunately. 

This hack was inspired by https://github.com/microsoft/vscode/issues/144417#issuecomment-1059979480.
